src/EdgeAnalysis.jl

```diff
@@ -22,4 +22,49 @@ end
 
 export TerminationTime
 export TerminationConversion
-export TerminationRateRatio
\ No newline at end of file
+export TerminationRateRatio
+
+"""
+Generate appropriate core Simulation/SystemSimulation object
+"""
+function tosim(sol,domains,inters,p)
+    return Simulation(sol,domains,inters,p)
+end
+
+"""
+Generate appropriate core Simulation/SystemSimulation object
+"""
+function tosim(sol,domains::Tuple,inters,p)
+   return SystemSimulation(sol,domains,inters,p) 
+end
+
+"""
+Generate appropriate edge Simulation/SystemSimulation object
+"""
+function getsim(inte,react,coreedgedomain,domain,inters,p,coretoedgespcmap)
```

Comment:
Input `domain` isn't used in this function? What is the difference between `coreedgedomain` and `domain`?

Reply:
Thanks! coreedgedomain is the domain containing the core and edge while domain is the one containing just the core.